charts/promtail/templates/prometheus-rules.yaml

```
@@ -0,0 +1,23 @@
+{{- if and .Values.serviceMonitor.enabled .Values.serviceMonitor.prometheusRule.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ template "promtail.fullname" . }}
```

As per official [Helm document](https://helm.sh/docs/chart_template_guide/named_templates/#the-include-function):

> It is considered preferable to use include over template in Helm templates simply so that the output formatting can be handled better for YAML documents.

```suggestion
  name: {{ include "promtail.fullname" . }}
```